packages/core/.storybook/manager.js

```diff
@@ -20,6 +20,15 @@ addons.register('storybook/ga-analytics', api => {
   }
 });
 
+addons.register('theme/reset', api => {
+  api.on(STORY_CHANGED, () => {
+    const body = document.querySelector('#storybook-preview-iframe').contentWindow.document.body;
+    body.removeAttribute('cds-theme');
+    body.classList.remove('cds-theme-light');
+    body.classList.remove('cds-theme-dark');
+  });
+});
+
```

Comment:
This will set the theme back to light when navigating between stories. There was buggy behavior when storybook would try to preserve a theme between navigation.